halysongoncalves/clappr-android
# Fix: Migrate fragment to fragment support

Goal
Change import fragment to use fragment android support.

How to Test
Run Unit Tests: ./gradlew clean test

1 - Open sample app
2 - Play video and ensure Media Control is working as expected
